src/components/ArticleCard.jsx

```diff
@@ -1,9 +1,12 @@
+import { useNavigate } from "react-router-dom"
+
 const ArticleCard = ({article}) => {
     const date = article.created_at
+    const navigate = useNavigate()
     return(<li key={article.article_id} className='article_card'> <h1>{article.title}</h1>
-    <p> By {article.author}, {date.slice(8, 10)}/{date.slice(5, 7)}/{date.slice(0, 4)}</p>
+    <p> By {article.author}, {date.slice(8, 10)}-{date.slice(5, 7)}-{date.slice(0, 4)}</p>
     <p>Topic: {article.topic}</p>
-    <button className="article_button">Read in full</button></li>)
+    <button className="article_button" onClick={(e)=>{navigate(`/articles/${article.article_id}`)}}>Read in full</button></li>)
```

Comment:
You don't need the e event parameter here as you don't use it, I'd just leave it empty.